README.md

```diff
@@ -1,137 +1,66 @@
+First, [setup your Bisq Seednode](https://github.com/bisq-network/bisq/tree/master/seednode#bisq-seed-node) so you have Tor, Bitcoin, and bisq-seednode running and fully synced. Then, enable BSQ data output on your Bisq Seednode with the following command
 
-#### Web Server
-```
-sudo apt-get update
-sudo apt-get install nginx
+```bash
+sudo sed -i -e 's!BISQ_DUMP_BLOCKCHAIN=false!BISQ_DUMP_BLOCKCHAIN=true!' /etc/default/bisq-seednode.env
+sudo service bisq-seednode restart
 ```
 
-Check that it's running
+It will take 10+ minutes before the seednode starts saving BSQ transaction data.
 
-```
-systemctl status nginx
-```
-Copy nginx.config to /etc/nginx/sites-available and symlink from /etc/nginx/sites-enabled. Remove sym link to
-default site. Replace `hostname` with site name.
+### Let's Encrypt
 
-Install certificate
+Next, request an SSL certificate for your server from Let's Encrypt using certbot-nginx
 
-```
-sudo apt-get install software-properties-common
-sudo add-apt-repository universe
-sudo add-apt-repository ppa:certbot/certbot
-sudo apt-get update
-sudo apt-get install python-certbot-nginx
-sudo certbot --nginx
+```bash
+sudo apt-get update -q
+sudo apt-get install -q -y nginx-core python-certbot-nginx
+sudo certbot --nginx --agree-tos --non-interactive -m ssl@example.com -d explorer.example.com
 ```
 
-Set automatic certificate renewal in crontab. Try to renew every month. Edit crontab...
+After obtaining your SSL certificate, you should be able to see the default page at https://explorer.example.com/
 
-```
-sudo crontab -e
-```
-...and paste this line2
-```
-0 0 1 * * certbot renew
-```
+### BSQ Indexer
 
-#### Explorer
-Some needed pyhton packages
-```
-sudo pip install python-bitcoinrpc
-sudo pip install simplejson
-sudo pip install gitpython
-```
-
-Setup variables for other commands below to use:
-```
-export EXPLORER_HOME=<your full path to explorer>
-export DATADIR=<path to bisq db directory>
-```
-
-examples:
-```
-#we assume you cloned this repo to: /home/user/bisq-explorer
-#we assume bisq-seednode was run with --appName=seed_2002 and testnet is used as network
-export EXPLORER_HOME=/home/user/bisq-explorer
-export DATADIR=/home/user/.local/share/seed_2002/btc_testnet/db
+Clone this repo to bisq user's homedir
+```bash
+curl -s https://raw.githubusercontent.com/bisq-network/bisq-explorer/master/install_bsq_explorer_debian.sh | sudo bash
 ```
 
-HTML need to point to explorer/www
-```
-cd /var/www
-sudo ln -s ${EXPLORER_HOME}/www html
-```
+### Nginx
 
-Copy Bisq seednode db directory contents to `$EXPLORER_HOME`
-```
-mkdir ${EXPLORER_HOME}/data
-cp -r ${DATADIR}/* ${EXPLORER_HOME}/data
+Install the nginx.conf from this repository, substituting explorer.example.com for your server hostname
+```bash
+sudo wget -O /etc/nginx/nginx.conf https://raw.githubusercontent.com/bisq-network/bisq-explorer/master/nginx.conf
+sudo sed -i -e '!__HOSTNAME__!explorer.example.com!g' /etc/nginx/nginx.conf
```

Comment:
I ran into an error when running this command:
`sed: -e expression #1, char 2: unknown command: `_'`

So instead i used:
`sudo sed -i -e 's/__HOSTNAME__/explorer.example.com/g' /etc/nginx/nginx.conf`

Reply:
Fixed in 1830d48